systems/plants/shapes/Element.cpp

```diff
@@ -46,6 +46,20 @@ namespace DrakeShapes
     return geometry != nullptr;
   }
 
+  void Element::getPoints(Eigen::Matrix3Xd &local_points)
+  {
+    if ( !hasGeometry() ) {
+      local_points = Eigen::Matrix3Xd();
+      return;
+    }
+    
+    Eigen::Matrix3Xd points;
+    geometry->getPoints(points);
+
+    Eigen::Matrix4Xd transformed_points = T_element_to_local * points.colwise().homogeneous();
+    local_points = transformed_points.colwise().hnormalized();
```

Comment:
This is fine if you're using Matrix4d as the transform type.

We've recently started switching to using Eigen's built in Transform class (in particular, the Isometry3d typedef). An Isometry3d is internally stored as just the first three rows of the transform; the last row is implicitly [0 0 0 1]. You can directly multiply an Isometry3d with a Vector3d or Matrix3Xd (i.e. using operator*), and have it do the rotation and translation.

But no need to change the Matrix4ds to Isometry3d right now I would say.